[31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
  execute_process(/usr/bin/python3 -m rosidl_adapter --package-name tm_msgs
  --arguments-file
  /home/rslomron/MoMa/tm_custom/build/tm_msgs/rosidl_adapter__arguments__tm_msgs.json
  --output-dir
  /home/rslomron/MoMa/tm_custom/build/tm_msgs/rosidl_adapter/tm_msgs
  --output-file
  /home/rslomron/MoMa/tm_custom/build/tm_msgs/rosidl_adapter/tm_msgs.idls)
  returned error code 1:

  Error with: tm_msgs SetPosition ---

  Traceback (most recent call last):

    File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
      return _run_code(code, main_globals, None,
    File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
      exec(code, run_globals)
    File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
      sys.exit(main())
    File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
      abs_idl_file = convert_to_idl(
    File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
      return convert_msg_to_idl(
    File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
      msg = parse_message_string(package_name, input_file.stem, content)
    File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 510, in parse_message_string
      raise InvalidFieldDefinition(line)

  rosidl_adapter.parser.InvalidFieldDefinition: ---

Call Stack (most recent call first):
  /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
  CMakeLists.txt:47 (rosidl_generate_interfaces)

[0m
